src/constructs/lambda/lambda.ts

```diff
@@ -129,6 +165,31 @@ export class GuLambdaFunction extends Function {
       });
     }
 
+    if (config.enabled) {
+      // For details, see https://github.com/guardian/ssm-to-env. Ultimately, it
+      // would be great to combine this with
+      // https://github.com/guardian/devx-config.
+
+      const deployToolsAccountId = StringParameter.fromStringParameterName(
+        this,
+        "deploy-tools-account-id-param",
+        NAMED_SSM_PARAMETER_PATHS.DeployToolsAccountId.path
+      );
+
+      const configPrefix = config.ssmPrefix ?? `/${scope.stage}/${scope.stack}/${this.app}`;
+
+      // TODO - guessing this needs to be parameterised by arch?
+      const configLayer = LayerVersion.fromLayerVersionArn(
+        scope,
+        "config-layer",
+        `arn:aws:lambda:eu-west-1:${deployToolsAccountId.stringValue}:layer:ssm-to-env-layer:6`
```

Comment:
Should this be region agnostic? Is `6` the version number? How do we update this over time?

Reply:
Region agnostic 👍 . Suggesting that version bumps are a PR change here for now and then a CDK version bump for the library user. But it might make sense to parameterise this in the pattern in the future - once we get more adoption perhaps and/or more versions.

Reply:
What about pulling the layer from S3 via [`code`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.LayerVersion.html#code)?

Reply:
The [lambda layer](https://github.com/guardian/ssm-to-env/tree/main) can be deployed via riff-raff which was updated to support deploying lambda layers, that will bump the version number when successfully deployed (the layer is configured to available organisation wide).  You could potentially store the targeted layer version in SSM and configure that here as with the deploy tools account number, or rely on a GuCDK version update to change the version used.